tensorboard/plugins/mesh/test_data/icosphere.ply

```diff
@@ -0,0 +1,189 @@
+ply

```

Comment:
Somehow this file was changed to have CRLF line endings. Please fix it
to use standard LF-only line endings.

Reply:
hm, I see LF-only (double checked by trying to match \r\n pattern). Are you sure it is not on your end only?

Reply:
I’m sure:

```
$ git clone -q https://github.com/podlipensky/tensorboard.git
$ cd tensorboard/
$ git checkout -q origin/tf-graphics-demo2
$ git rev-parse --verify HEAD
1073a1be70a431208f2694e9571e38de4196f3db
$ # normal file, LF only:
$ head -n 1 tensorboard/plugins/mesh/README.md | xxd
00000000: 2320 4d65 7368 2050 6c75 6769 6e0a       # Mesh Plugin.
$ # this PLY file, CRLF:
$ head -n 1 tensorboard/plugins/mesh/test_data/*.ply | xxd
00000000: 706c 790d 0a                             ply..
$ file tensorboard/plugins/mesh/test_data/*.ply
tensorboard/plugins/mesh/test_data/icosphere.ply: ASCII text, with CRLF line terminators
```

Furthermore, if you use `git show af14d2e2 '*/icosphere.ply'` to inspect
the commit in which the file was introduced, you will see that the diff
lines contain carriage returns, indicated by `^M` symbols:

```
--- /dev/null
+++ b/tensorboard/plugins/mesh/test_data/icosphere.ply
@@ -0,0 +1,189 @@
+ply^M
+format ascii 1.0^M
```

Reply:
Please apply the following patch (`git am 0001-*.txt`) and push it to
your branch:

[0001-mesh-remove-carriage-returns-from-test-data.txt](https://github.com/tensorflow/tensorboard/files/3254533/0001-mesh-remove-carriage-returns-from-test-data.txt)